src/components/footer.tsx

```diff
@@ -0,0 +1,147 @@
+import Image from "next/image";
+import { Button } from "./ui/button";
+import Link from "next/link";
+
+export default function Footer() {
+  const footerMenuList = [
+    {
+      header: "KALKULAČKY",
+      items: [
+        {
+          link: "/wages#net-salary",
```

Comment:
Tieto linky si vytiahni do nejakého lepšieho objektu, niečo takéto:
```
const PATHS = {
    WAGES: {
        link: "/wages"
        description: "SZČO"
    }
}
```
Ideálne si to ulož do nejakého helpers/utils foldra, lebo to budeš používať všade v aplikácii aby si nemusel natvrdo triafať stringy.

Reply:
Tým SCREAMING casom dávaš najavo, že je to nejaká nemenná konštanta. Ja zvyknem už tie samotné konkrétne atribúty písať camelCasom, ale to už nieje nejak podstatné.

Reply:
Keďže ale vo footeri potrebuješ array tých linkov a nie iba jeden objekt, vieš si to vytiahnuť takto
```
const PATHS_OPTIONS = Object.values(PATHS)
```
Ak to budeš používať ešte niekde inde, tak si to kludne ulož tým SCREAMING casom niekde v utils súbore, ak nie tak si to nechaj iba v footeri.